parser-typechecker/src/Unison/Codebase/Causal.hs

```diff
@@ -96,7 +96,16 @@ data Causal m h e
           , tails :: Map (RawHash h) (m (Causal m h e))
           }
 
-Lens.makeLensesFor [("head", "head_")] ''Causal
+-- | Focus the current head, keeping the hash up to date.
+head_ :: Hashable e => Lens.Lens' (Causal m h e) e
```

Comment:
The previous lens was a bit unsafe since you'd end up with a causal whose hash doesn't match its contents; which caused a bug that I had to track down to here 😓